initramfs/scripts/local-bottom/fixrtc-mount

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh -e
+# initramfs local-bottom script for fixrtc-mount
+
+PREREQ=""
+
+# Output pre-requisites
+prereqs()
+{
+        echo "$PREREQ"
+}
+
+case "$1" in
+    prereqs)
+        prereqs
+        exit 0
+        ;;
+esac
+
+# This script looks at root file system files to get an accurate as possible
+# date. It runs when we already have a mounted root, as opposed to the 'fixrtc'
+# script, which uses the last mount time of the root file system. In devices
+# where there is no battery for the RTC, 'fixrtc' is not effective, as the mount
+# time will always be inaccurate. This script helps in those cases.
+
+BROKEN_CLOCK=""
+
+# shellcheck disable=SC2013
+for x in $(cat /proc/cmdline); do
+        case ${x} in
+        fixrtc)
+                BROKEN_CLOCK=1
+        ;;
+        esac
+done
+
+if [ -n "$BROKEN_CLOCK" ]; then
+        snapsfolder=/root/writable/system-data/var/lib/snapd/snaps
+
+        current=$(date +%s)
+
+        # We start with date at the time of writing this
+        newdate=$(date -d "2017-06-09" +%s)
+
+        if [ -d "$snapsfolder" ]; then
+                otherdate=$(stat -c %Y "$snapsfolder")
+                if [ "$otherdate" -gt "$newdate" ]; then
+                        newdate=$otherdate
+                fi
+        fi
+
+        if [ "$newdate" -gt "$current" ]; then
+                date -s "@$newdate"
```

Comment:
Off by one space here

Reply:
Thanks, there was a <tab> in there, fixed now.